tests/page-objects/contacts/contacts.wdio.page.js

```diff
@@ -213,6 +221,47 @@ const editDistrict = async (districtName, editedName) => {
   await (await genericForm.submitButton()).click();
 };
 
+const createNewAction = async (formName) => {
+  await (await newActionContactButton()).waitForDisplayed();
+  await (await newActionContactButton()).click();
+  await openForm(formName);
+};
+
+const openForm = async (name) => {
+  // this is annoying but there's a race condition where the click could end up on another form if we don't
+  // wait for the animation to finish
+  await (await $('.action-container .detail-actions #relevant-contacts-form')).waitForDisplayed();
+  await browser.pause(50);
+  for (const form of await forms()) {
+    if (await form.getText() === name) {
+      await form.click();
+      await (await formTitle()).waitForDisplayed();
+      return;
+    }
+  }
+  throw new Error(`Form with name: "${name}" not found`);
+};
+
+const openReport = async () => {
+  await (await rhsReportListElement()).waitForDisplayed();
+  await (await rhsReportListElement()).click();
```

Comment:
This is already an async function, meaning it's going to return a promise automatically. If you `await` it here, that'll just wait on the promise to get a real value, and then this whole function will wrap that value in another promise for the return value. Better to just return the promise you have in the first place.

```suggestion
  return (await rhsReportListElement()).click();
```

Please make a similar change to all of the following functions that do similar.